.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -7,6 +7,8 @@ body:
   - type: markdown
     attributes:
       value: Thanks for taking the time to fill out this bug report!
+      Please do not report issue on 5.2.1 version, this version is not maintained anymore.
+      Only issues on version > V6 will be handled. Please also ensure your issue is reproduced with the last release!
 
   - type: textarea
```

It's less confusing to use smaller input for version
```suggestion
  - type: input
```

---

> It's less confusing to use smaller input for version
> ```suggestion
>   - type: input
> ```

I agree but I don't think this is a good way to proceed :) putting an input will make necessary to patch this file to include each new release! You put input in the past but it leads to '6.0.0-beta' input, as there were 10 beta, these tickets are confusing now. A free field allows you to put the exact version

---

I agree, but input is smaller textarea (also allow for free text)